src/database/queries/order.query.js

```diff
@@ -43,4 +43,40 @@ const getOrdersByRestaurantId = async (restaurantId, orderStatus) => {
   return orders;
 };
 
-export { getOrdersByRestaurantId };
+const createOrder = async (
+  customerId,
+  restaurantId,
+  specifications,
+  checkoutPrice,
+  addressId
+) => {
+  const newOrder = await sql`
+    INSERT INTO orders (
+      customer_id,
+      restaurant_id,
+      specifications,
+      checkout_price,
+      address_id,
+      order_status
+    ) VALUES (
+      ${customerId}, 
+      ${restaurantId}, 
+      ${specifications || null}, 
```

Comment:
make it default value in the function parameter 
```
const createOrder = async (
  customerId,
  restaurantId,
  specifications = null,
  checkoutPrice,
  addressId
) => {
...
```

Reply:
Done